pkg/rtc/transport.go

```diff
@@ -1370,6 +1370,11 @@ func (t *PCTransport) postEvent(event event) {
 
 func (t *PCTransport) processEvents() {
 	for event := range t.eventCh {
+		if t.isClosed.Load() {
+			// just drain the channel with processing events
+			continue
```

Comment:
not sure if there is a need to `continue` here. Could have just done a `break`. But, I was not sure if a channel with entries in it can hold on to some references and leak memory. I don't think so, but doing a `continue` to drain the channel seemed fine. There should be not more than a few events to drain at most.